airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -791,7 +1269,8 @@ definitions:
         type: string
         enum: [JsonDecoder]
   ListPartitionRouter:
```

Comment:
Are there some interpolation contexts missing for the list partition router?

Reply:
yes, added them